cmplibrary/cmp-consumer-proguard-rules.pro

```diff
@@ -0,0 +1,53 @@
+# Proguard rules for CMP consumers
```

Comment:
@bohdan-go-wombat by any chance, is it possible to change the cmp's log level in consumer release using proguard rules file?

Reply:
@Nevazhnovu the proguard rules aren’t related with the SDK log messages

Reply:
@Nevazhnovu yes, @carmelo-iriti is right, the logs are not related to the proguard rules, but can you elaborate more on what you wanted? maybe we can figure it out, but we need more context on what we want to achieve